app/ui-master/client/main.js

```diff
@@ -129,12 +129,27 @@ Template.body.onRendered(function() {
 	});
 });
 
-Template.main.onCreated(function() {
+Template.main.onCreated(async function() {
 	tooltip.init();
-});
 
+	this.userReady = new ReactiveVar(false);
```

Comment:
could you keep the `skipActiveUsersToBeReady`?
maybe use this: `this.userReady = new ReactiveVar(skipActiveUsersToBeReady);`